tools/main.py

```diff
@@ -137,9 +138,9 @@ def main():
     if cfg.model.load_weights and check_isfile(cfg.model.load_weights):
         if cfg.model.pretrained and not cfg.test.evaluate:
             state_dict = torch.load(cfg.model.load_weights)
-            model.load_pretrained_weights(state_dict)
+            load_model(model, pretrained_dict=state_dict)
```

Comment:
This if is seems to be useless in here. I propose to leave `load_pretrained_weights(model, cfg.model.load_weights)` from the second branch